src/twilio/tests/Readme.md

```diff
@@ -1,8 +1,10 @@
 # Ballerina Twilio Endpoint - Tests
 
-The Twilio connector allows you to send SMS, voice, and OTP messages through the Twilio REST API. You can also send
+The Twilio connector allows you to send SMS, voice, OTP, and WhatsApp messages through the Twilio REST API. You can also send
 user secrets via SMS or voice message, verify OTP, and add and delete users. It handles basic authentication.
 
+WhatsApp messaging requires users to use predefined [message templates](https://www.twilio.com/docs/sms/whatsapp/tutorial/send-whatsapp-notification-messages-templates). Templates are message formats which can be used over and over again to message users. Use of templates enables to maintain high-quality content as well as to avoid spam messages. However, this requires the WhatsApp message templates to be predefined.
```

Comment:
```suggestion
WhatsApp messaging requires users to use predefined [message templates](https://www.twilio.com/docs/sms/whatsapp/tutorial/send-whatsapp-notification-messages-templates). Templates are message formats, which can be used over and over again to message users. Use of templates enables to maintain high-quality content as well as to avoid spam messages. However, this requires the WhatsApp message templates to be predefined.
```

Reply:
Done